src/open_inwoner/cms/tests/cms_tools.py

```diff
@@ -121,14 +122,13 @@ def create_apphook_page(
         extension_args["extended_object"] = p
         CommonExtension.objects.create(**extension_args)
 
-    # TODO find out why this doesn't work
     # create app_config
-    # if hook_class.app_config:
-    #     # attach it to the page for convenience
-    #     if config_args is None:
-    #         config_args = dict()
-    #     config_args["namespace"] = hook_class.app_name
-    #     p.app_config = hook_class.app_config.objects.create(**config_args)
+    if app_config := apphook_pool.get_apphook(hook_class.__name__).app_config:
+        # attach it to the page for convenience
+        if config_args is None:
+            config_args = dict()
+        config_args["namespace"] = hook_class.app_name
+        p.app_config = app_config.objects.get_or_create(**config_args)
```

Comment:
Cooool. Didn't know about the apphook_pool. Could be something for the hacky stuff in open_inwoner/cms/utils/page_display.py